README.md

```diff
@@ -14,16 +14,38 @@ heroku buildpacks:add https://github.com/moneymeets/python-poetry-buildpack.git
 heroku buildpacks:add heroku/python
 ```
 
-Generation of the `runtime.txt` can be skipped by setting `DISABLE_POETRY_CREATE_RUNTIME_FILE` to `1`:
+## Configuration
+
+### Python
+
+Python version can be forced by setting `PYTHON_RUNTIME_VERSION` variable:
 
 ```
-heroku config:set DISABLE_POETRY_CREATE_RUNTIME_FILE=1
+heroku config:set PYTHON_RUNTIME_VERSION=3.9.1
 ```
 
-If `DISABLE_POETRY_CREATE_RUNTIME_FILE` is set, required Python version can be specified in `runtime.txt`. Otherwise, if `runtime.txt` is present in the repository, the buildpack will prevent the app from being deployed in order to avoid possible ambiguities.
+### Poetry
 
-The Poetry version can be specified by setting `POETRY_VERSION` in Heroku config vars. Otherwise it will default to a hard coded version.
+The Poetry version can be specified by setting `POETRY_VERSION` in Heroku config vars. Otherwise, it will default to a hard coded version.
 
 ```
 heroku config:set POETRY_VERSION=1.1.0
 ```
+
+Generally all variables starting with `POETRY_` are passed on to Poetry by this buildpack; see [Poetry documentation](https://python-poetry.org/docs/configuration/#using-environment-variables) for possible uses.
+
+Exporting of development dependencies (e.g. to run tests in CI pipelines) can be optionally enabled by setting  `POETRY_EXPORT_DEV_REQUIREMENTS` to `1`:
+
+ ```
+ heroku config:set POETRY_EXPORT_DEV_REQUIREMENTS=1
+ ```
+
+### runtime.txt
+
+Generation of the `runtime.txt` can be skipped by setting `DISABLE_POETRY_CREATE_RUNTIME_FILE` to `1`:
+
+```
+heroku config:set DISABLE_POETRY_CREATE_RUNTIME_FILE=1
+```
+
+If `DISABLE_POETRY_CREATE_RUNTIME_FILE` is set, required Python version can be specified in `runtime.txt`. Otherwise, if `runtime.txt` is present in the repository, the buildpack will prevent the app from being deployed in order to avoid possible ambiguities.
```

Comment:
```suggestion
If `DISABLE_POETRY_CREATE_RUNTIME_FILE` is set, the required Python version can be specified in `runtime.txt`. Otherwise, if `runtime.txt` is present in the repository, the buildpack will prevent the app from being deployed in order to avoid possible ambiguities.
```

Reply:
Committed.